src/nvim/api/win_config.c

```diff
@@ -160,6 +160,9 @@
 ///   - noautocmd: If true then no buffer-related autocommand events such as
 ///                  |BufEnter|, |BufLeave| or |BufWinEnter| may fire from
 ///                  calling this function.
+///   - fixed: When anchor is NW or SW and wrap is off and the float window would be
```

Comment:
I would rephrase this as putting what the option enables upfront. something like: the position is kept fixed even if the window would be truncated.